source/renderer/app/components/wallet/settings/ICOPublicKeyQRCodeDialog.js

```diff
@@ -11,14 +11,14 @@ import Dialog from '../../widgets/Dialog';
 import iconCopy from '../../../assets/images/clipboard-ic.inline.svg';
 import styles from './PublicKeyQRCodeDialog.scss';
 import globalMessages from '../../../i18n/global-messages';
-import type { ReactIntlMessage } from '../../../types/i18nTypes';
+
+import { messages } from './ICOPublicKeyQRCodeDialog.messages';
```

Comment:
The storybook book related to `ICOPublicKeyQRCodeDialog` is broken on develop branch:
https://www.chromatic.com/test?appId=61afa6b2acfbf7003ae248a3&id=61afab78e0748c003abdf55b

Reply:
Thank you, @renanvalentin, you fixed that in this branch!

@dmitrii-gaico would you pls just check the ICO PublicKey QR Code dialog is still working? thank you :)